### ISSUE TITLE
Argument 2 passed to Magento\CloudComponents\Model\Cache\Evictor::run() must be of the type int, string given

### ISSUE THREAD

**Issue by DanReinders** [magento/magento-cloud-components#47]
```
php bin/magento cache:ev
Begin scanning of cache keys
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Magento\CloudComponents\Model\Cache\Evictor::run() must be of the type int, string given, called in /app/by3va3srrs2zq/vendor/magento/magento-cloud-components/Model/Cache/Evictor.php on line 79 and defined in /app/by3va3srrs2zq/vendor/magento/magento-cloud-components/Model/Cache/Evictor.php:95
Stack trace:
#0 /app/by3va3srrs2zq/vendor/magento/magento-cloud-components/Model/Cache/Evictor.php(79): Magento\CloudComponents\Model\Cache\Evictor->run('localhost', '6370', 1)
#1 /app/by3va3srrs2zq/vendor/magento/magento-cloud-components/Console/Command/CacheEvict.php(50): Magento\CloudComponents\Model\Cache\Evictor->evict()
#2 /app/by3va3srrs2zq/vendor/symfony/console/Command/Command.php(255): Magento\CloudComponents\Console\Command\CacheEvict->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /app/by3va3srrs2zq/vendor/magento/framework/Interception/Interceptor.php(58): Symfony\Component\Console\Command\Command in /app/by3va3srrs2zq/vendor/magento/magento-cloud-components/Model/Cache/Evictor.php on line 95
```

Magento Commerce Cloud
"magento/magento-cloud-metapackage": ">=2.3.5 <2.3.6"

Command:
php bin/magento cache:evict

Result is the PHP fatal error show above.

app/etc/env.php:
```
<?php
return array (
  'cache' =>
  array (
    'frontend' =>
    array (
      'default' =>
      array (
        'backend' => 'Cm_Cache_Backend_Redis',
        'backend_options' =>
        array (
          'server' => 'localhost',
          'port' => '6370',
          'database' => 1,
        ),
      ),
      'page_cache' =>
      array (
        'backend' => 'Cm_Cache_Backend_Redis',
        'backend_options' =>
        array (
          'server' => 'localhost',
          'port' => '6370',
          'database' => 2,
        ),
      ),
    ),
  ),
...
```

**Comment by DanReinders** [comment]
During a deployment to Magento Cloud the env.php file is regenerated by the system.

'port' => '6370',

in the above the value of ['port'] is a string not an integer as required by the method's type hint:
```    /**
     * @param string $host
     * @param int $port
     * @param int $db
     * @return int
     */
    private function run(string $host, int $port, int $db): int
```

**Comment by YPyltiai** [comment]
@NadiyaS , @oshmyheliuk FYI

**Comment by oshmyheliuk** [comment]
@DanReinders which version of `magento-cloud-components` do you have installed, this error was fixed in the latest version.
https://github.com/magento/magento-cloud-components/blob/develop/Model/Cache/Evictor.php#L76-L80

**Comment by DanReinders** [comment]
I have 1.0.6 installed.  Yes, issue looks fixed in 1.0.7.

Closing this issue.  Thank you.
